js/dimensions.js

```diff
@@ -40,13 +40,15 @@ $(document).ready(function() {
 
     this.showResult = function() {
       $uploadCrop.result({
-        type: 'canvas', 
+        type: 'blob',
```

Comment:
Needed to use a blob object to handle larger images. More details in issue: https://github.com/chapmanu/dimensions/issues/21